signingscript/task.py

```diff
@@ -213,6 +213,19 @@ def _check_scopes_exist_and_all_have_the_same_prefix(scopes, prefixes):
     return output
 
 
+def _get_signing_function_from_format(format):
```

Comment:
I really like this approach. The only problem I see is that we need to rename the scope in puppet + TC clients from `autograph_fennec_sha` to `autograph_apk_fennec_sha1` and possible elsewhere. But since this is not yet fully deployed, I think it'll be easy!

Also, I love the `_apk` substring, makes more sense to separate from possible future formats that we gradually switch to Autograph.